mlir/include/mlir/Dialect/LLVMIR/LLVMDialect.h

```diff
@@ -33,6 +33,7 @@
 #include "mlir/Support/ThreadLocalCache.h"
 #include "llvm/ADT/PointerEmbeddedInt.h"
 #include "llvm/IR/DerivedTypes.h"
+#include "llvm/IR/InstrTypes.h"
```

Comment:
It's required here because `llvm::OperandBundleDefT` appears in the generated headers. Or we could replace it with a forward declaration to `llvm::OperandBundleDefT`?

Reply:
Ah I see now. So let's keep it.